pkg/streamingpromql/types/fpoint_ring_buffer.go

```diff
@@ -161,7 +161,6 @@ func (b *FPointRingBuffer) Reset() {
 
 // Close releases any resources associated with this buffer.
 func (b *FPointRingBuffer) Close() {
-	b.Reset()
```

Comment:
Why is the `Reset` call unnecessary? Which practical consequences does removing it have?

Reply:
Calling `Reset` is unnecessary because a `FPointRingBuffer` isn't expected to be used after `Close` is called.